src/Routes.js

```diff
@@ -0,0 +1,17 @@
+import { Routes, Route, Navigate } from "react-router-dom";
+import { WishList, Products, Cart, LoginPage } from "./Components";
+import { PrivateRoute } from "./PrivateRoute";
+export const RouterComponent = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<Products />} />
+      <PrivateRoute path="/wishlist" element={<WishList />} />
+      <PrivateRoute path="/cart" element={<Cart />} />
+      <Route path="/login" element={<LoginPage />} />
+      <Route path="*" element={<RouteNotFound />} />
+    </Routes>
+  );
+};
+const RouteNotFound = () => {
```

Comment:
You could show a 404 page instead of navigating to products page - that would be better UX I guess as it might create a confusion why it's not going to the route that user wants to visit.